openqa-investigate

```diff
@@ -374,7 +374,13 @@ investigate() {
     comment_id=$(sync_via_investigation_comment "$id" "$first_cluster_job_id") || rc=$?
     [[ $rc != 255 ]] && return $rc
 
-    out=$(trigger_jobs "$id" "${@:2}") || return $?
+    out=""
```

Comment:
This could be 
`out=${$(trigger_jobs "$id" "${@:2}"):-""}`

I think it would simplify the logic a bit?

Reply:
I find this less readable, I prefer to split the assignation to be more clear on the intent and the expectation.